### DISCUSSION TITLE
Should these kata be considered duplicates?

### DISCUSSION THREAD

**Discussion by XRFXLP** [codewars/content-issues#14] [Q&A] [answered]
Here's a recently published kata: [Find the smallest solution](https://www.codewars.com/kata/5fc2b40107eafd000d3c0b47/) which is same problem as 2017: [Simple Fun #243: Perfect Team Of MinimalSize](https://www.codewars.com/kata/590a924c7dfc1a238d000047/) should this be considered duplicate?

**Comment by hobovsky** [comment, marked as answer]
I have not solved any of them, but reading through the description I think they are almost identical. Even input ranges and rank votes seem to be identical :)

I asked the author to unpublish and provide a translation instead.
